combiners/SSNet_predictions_notebook.ipynb

```diff
@@ -0,0 +1,1160 @@
+{
```

Comment:
Please verify if this file is similar to SSNet_predictions.py. 
If there are some good tweaks or changes, you can use SSNet_predictions.py only and remove this notebook.